scripts/webdriverioTester.js

```diff
@@ -24,6 +24,14 @@ const argv = require('minimist')(process.argv.slice(2), {
 })
 const TOKEN_REVOKED = '~'
 
+const GitHubAPI = require('github')
+const github = new GitHubAPI({
```

Comment:
It's a bit creepy to have a module-level global hanging out here. Can't you instantiate this thing somewhere else? A dev doesn't expect something to be instantiated as part of an `import` statement